wfl/calculators/espresso.py

```diff
@@ -140,9 +84,6 @@ def calculate(self, atoms=None, properties=_default_properties, system_changes=a
             calculation_succeeded = True
             if 'DFT_FAILED_ESPRESSO' in atoms.info:
                 del atoms.info['DFT_FAILED_ESPRESSO']
-            if "__calculator_output_prefix" in atoms.info:
```

Comment:
The calculator only modifies the calculator kwargs, not configs themselves, so I have removed this check. Or should we keep it for completeness/future uses?

Reply:
I'm ambivalent. On the one hand, it isn't necessary, as you noted.  On the other hand, it doesn't do any harm, and I wonder if we should just leave it in there in case some later version _does_ change the configs (like the Vasp calculator does).

Reply:
Let's keep this bit of code. I've undid my previous removal.